themes/ddbasic/sass/components/ting-subsearch.scss

```diff
@@ -0,0 +1,130 @@
+// ting_subsearch styling.
+
+@import '../base.scss';
+
+.ting-subsearch-message-wrapper {
+  background-color: $grey-light;
+  padding-top: 30px;
+
+  .ting-subsearch-message {
+    max-width: 1124px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+}
+
+.message-box {
+  padding: 15px 12px;
+
+  .message-box--title {
+    margin-bottom: 12px;
+  }
+
+  .message-box--list-item {
+    margin-bottom: 5px;
+    padding: 10px 25px 10px 10px;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    align-content: space-between;
+    min-height: 56px;
+    background-color: $grey-light;
+    overflow: hidden;
+
+    .message-box--content {
+      position: absolute;
+      overflow: hidden;
+      .field-name-ting-title h2 {
+        font-size: 100%;
+        font-weight: bold;
+        margin-bottom: 0;
+        line-height: unset;
+      }
+
+      @media screen and(max-width: 1200px) {
+        position: unset;
+      }
+    }
+
+    .message-box--see-more {
+      transition: background-color .3s cubic-bezier(.165,.84,.44,1),color .3s cubic-bezier(.165,.84,.44,1);
+      background-color: $grey;
+      color: $charcoal;
+      height: 36px;
+      padding: 10px 16px;
+      border-radius: 5px;
+      margin-left: auto;
+      z-index: 1;
+      box-shadow: -20px 0px rgba(241, 242, 242, 1);
+
+      &:hover {
+        background-color: $grey-dark;
+        color: $white;
+      }
+
+      @media screen and(max-width: 1200px) {
+        display: none;
+      }
+    }
+  }
+
+  .message-box--external-search {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .message-box--external-search a {
+    position: relative;
+    display: inline-block;
+    text-align: center;
+    margin-top: 15px;
+    padding: 12px 25px 7px 25px;
+    background-color: #000;
+    border-radius: 5px;
+    color: $color-text-on-secondary;
+
+    &:hover {
+      color: $color-text-on-secondary;
+    }
+  }
+
+  .field-name-ting-type {
+    @media screen and(max-width: 970px) {
+      display: none !important;
+    }
+  }
+}
+
+div.search-field-in-content--message-tss {
+  width: 100%;
+  float: left;
+  border: none;
+  padding: 20px 15px;
+  background-color: $color-primary;
+  color: $color-text-on-primary;
+
+  .after & {
+    margin-top: 30px;
+  }
+
+  .before & {
+    margin-bottom: 30px;
+  }
+
+  a,
+  a.active {
+    &:hover {
+      color: $charcoal;
+    }
+  }
+}
+
+.pane-search-result .pane-content .message-box--title strong {
```

Comment:
Can we do without this very specific selector?

Reply:
I relocated this rule under the other styling. Had to add a `div.` to the selector to make it specific enough.

Some other place in the theme it is setting a too big font size for strong elements.